frontend/src/photoView/photoView.js

```diff
@@ -11,11 +11,135 @@ const SIDES = {
     BINDER: 'binder',
 };
 
+export class FVDisplayWidget extends React.Component {
```

Comment:
`FVDisplayWidget` -- didn't know what `FV` was when I first read this. Plz rename to something more explicit / not abbreviated so other readers will know.